htdocs/core/menus/standard/eldy.lib.php

```diff
@@ -2157,6 +2157,7 @@ function get_left_menu_mrp($mainmenu, &$newmenu, $usemenuhider = 1, $leftmenu =
 			$newmenu->add("", $langs->trans("MenuMRP"), 0, $user->hasRight('mrp',  'read'), '', $mainmenu, 'mo', 0, '', '', '', img_picto('', 'mrp', 'class="paddingright pictofixedwidth"'));
 			$newmenu->add("/mrp/mo_card.php?leftmenu=mo&amp;action=create", $langs->trans("NewMO"), 1, $user->hasRight('mrp',  'write'), '', $mainmenu, 'mo');
 			$newmenu->add("/mrp/mo_list.php?leftmenu=mo", $langs->trans("List"), 1, $user->hasRight('mrp',  'read'), '', $mainmenu, 'mo');
+			$newmenu->add("/mrp/mo_list_assignable.php?leftmenu=mo", $langs->trans("AssignableList"), 1, $user->hasRight('mrp',  'read'), '', $mainmenu, 'mo');
```

Comment:
I don't understand what is goal of this page.
What means qty missing ? Missing for what ?

Reply:
The Page views all MO's with the state (validated, in production, produced) where products can consuming.

**Currently:**
After a some stock movement, the warehouse clerk must go on all MO's  MO -> Production then view if its possible to consume a Product.
This is not practicable if you have more then 20 running MO´s. As example in our manufacturing industry (engineering company) we have about 500-1000 MO´s in production at the same time. This is normal in a manufacturing industry.

**With the View:**
After a some stock movement, the warehouse clerk go on the new view and see all MO's where Products can assigne/consume.

"What means qty missing ? Missing for what ?"
=> this means how many quantity of a Product is missing, to reach a complete consuming for a MO-Line.

I hope, i could tell the problem explain well. English is a bit problematical for me ;).

Reply:
Think you the naming "assign" is bad?
If it's better for the naming in Dolibarr, i can rename all to "consume"?

Reply:
To give my thoughts. Maybe do the same like column 'shippable' in customer order by adding column 'producible' in MO list. But do not throw away this new page but make like this PR for customer order: https://github.com/Dolibarr/dolibarr/pull/21596

Reply:
Hi @fappels,
the shippable in the sales orders work different.
![image](https://user-images.githubusercontent.com/78662388/210597385-71b314e3-15ec-4b34-b839-7a0a615deea3.png)

**Example for the new view:**
- A new Item receipt from a purchase order is booked.
- The clerk looks in the new view and filter the view after the product´s from the item receipt, he find instand the MO's and can consume thir products to the MO.

**Other example:**
- A project need to focus in the company.
- The clerk looks in the new view and filter all MO's where can consume for a defined Project.

P.S. You idea to have a state "complete consumable" is a nice feature for the next step for this PR.
P.P.S. A similar view for sales order is also a good idea, i thought about that too. But in the feature the view's have to much different workflows. Let's think on Services, Workstations and so on.